src/components/form/Input.tsx

```diff
@@ -114,7 +115,7 @@ export const Input = ({
         multiline && device.platform === 'ios' && styles.multiline,
         !isValid && !!errorMessage && styles.inputError
       ]}
-      errorStyle={styles.inputError}
+      errorStyle={[styles.inputError, !errorMessage && styles.inputErrorHeight]}
```

Comment:
why were this changes with heights added?

i can see a difference in the paddings between some form inputs/label now:

|before|after|
|---|---|
|![IMG_87AE59423535-1](https://user-images.githubusercontent.com/1942953/222216648-64ce86f4-380a-4641-a884-7109a8b81de0.jpeg)|![Simulator Screen Shot - iPhone 13 Pro Max - 2023-03-01 at 18 28 39](https://user-images.githubusercontent.com/1942953/222216681-04315521-55ec-4c5e-8393-f8384b1ed1ad.png)|

you can see the differences between the first one and at the last ones.

Reply:
I have found the cause of the problem and I will fix it. I will edit the size of the input again.

Reply:
increased the size of the input container from `normalize(65)` to `normalize(75)`

https://github.com/smart-village-solutions/smart-village-app-app/commit/5abeb5b34d774762c238b6dda8492f996627c635